srml/balances/src/lib.rs

```diff
@@ -170,6 +170,12 @@ mod tests;
 
 pub use self::imbalances::{PositiveImbalance, NegativeImbalance};
 
+pub const DEFAULT_EXISTENTIAL_DEPOSIT: u32 = 0;
```

Comment:
Sorry :sweat_smile:, but why don't you use directly `paramter_types!` here?

Reply:
because the type is Balance, and within the module we don't know what the type of Balance is, so people will have to define their own parameter types and then do into.

Reply:
make sense?